src/providers/aws.rs

```diff
@@ -109,6 +109,38 @@ use std::time;
 use tracing::instrument;
 use tracing_futures::Instrument;
 
+/// Dictate how a set of instances should be launched.
+#[derive(Debug, Clone)]
+#[allow(missing_copy_implementations)]
```

Comment:
What's wrong with having it be copy?

Reply:
In general, making types `Copy` is a backwards-compatibility risk. It means that if we _later_ add a field that can't be `Copy` (like a `String`), then we'd have to do a breaking release. In general, you should only implement `Copy` for public types if you think it is highly unlikely that they might gain non-`Copy` members in the future. For this, I think there's a decent chance we may add a `String` field later, so no `Copy` :)